package java_study.chapter10;

public class CatchByExceptionKindExample {

	public static void main(String[] args) {

		try {
			System.out.println(args[0]);
			System.out.println(args[1]);

			int value1 = Integer.parseInt(args[0]);
			int value2 = Integer.parseInt(args[1]);
			int result = value1 + value2;
			System.out.println(result);
		} catch (Exception e) {
			System.out.println(e);

		} finally {
			System.out.println("다시 실행하세요.");
		}
	}

}
